person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :id, :name, :age, :parent_permission
+
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    true if @age >= 18 || @parent_permission
+  end
```

Comment:
- [ ] In this method, it would be better if you use **of_age?** method to check if age is greater than or equal to 18 to follow the requirements and to make your code DRY :+1:
```

  def can_use_services?
    of_age?() >= 18 || @parent_permission
  end
```

Reply:
@HAMMAS-SALEEM  thanks 
fixed